lib/widgets/theme.dart

```diff
@@ -158,6 +159,7 @@ class DesignVariables extends ThemeExtension<DesignVariables> {
       groupDmConversationIconBg: const Color(0x33cccccc),
       loginOrDivider: const Color(0xff424242),
       loginOrDividerText: const Color(0xffa8a8a8),
+      mutedUnreadBadge: const HSLColor.fromAHSL(0.5, 0, 0, 0.3).toColor(),
```

Comment:
This is pretty low-contrast in dark mode, but as with many of our UI
    elements, we'll adjust as necessary when we have a design from Vlad.

From the screenshot, this gets pretty hard to see. It's good to have this properly wired up into DesignVariables, but I think the existing value that's the same as in light theme actually works out better.

Let's maybe just make this the same value (with 0.8 lightness) as for the light theme, and leave a `// TODO(design-dark)` for it. Or spend a minute or two trying values in between like 0.6, and pick something that looks clear.